targets/objc/make.js

```diff
@@ -282,10 +282,7 @@ function GetArrayPropertyDeserializer(property, datatype, api) {
     else if (property.isclass) {
         getter = "[[" + api.name +  property.actualtype + " new] initWithDictionary:[member_list objectAtIndex:i]]";
     }
-    else if (property.isenum) {
```

Comment:
I didn't fully investigated this, but I found some issues.
1 there are three more "else if (property.isenum)".
2. the enum type must call conversion  getter api(intValue or boolValue) to get real data from number object if it is required enum value.
3. the current implementation  is used this code to generate getter type, so I worry about the adding getting api and returning enum value directly can make some compile errors too if it is used to put the value is set to object.